x/auth/module.go

```diff
@@ -31,7 +31,7 @@ import (
 )
 
 // ConsensusVersion defines the current x/auth module consensus version.
-const ConsensusVersion = 5
+const ConsensusVersion = 4
```

Comment:
This comment made me re-check everything: https://github.com/cosmos/cosmos-sdk/pull/14482#discussion_r1062276855 and this was bumped with the params changes already.